packages/@lwc/integration-karma/test/light-dom/scoped-slot/index.spec.js

```diff
@@ -61,7 +61,7 @@ describe('scoped slots', () => {
         // For standard slot content, "slot" attribute goes directly on the element unlike scoped
         //  slots where the attribute goes on the template tag
         expect(child.querySelector('.slotname3').innerHTML).toBe(
```

Comment:
Run the nucleus tests through all tests, even the non-white listed and verify that it doesn't break anything